content/en/docs/contributing/localization.md

```diff
@@ -415,9 +415,16 @@ their own locale-specific PRs.
 
 ### Purely editorial changes across locales are OK {#patch-locale-links}
 
-**Purely editorial** page updates such as fixing broken link paths can span
-locales. A purely editorial change is one that **does not** impact the meaning
-of the page content.
+**Purely editorial** page updates are changes that **do not** affect the
+existing content and can span multiple locales. These include:
+
+- **Link maintenance**: Fixing broken link paths when pages are moved or
+  deleted.
+- **Targeted content additions**: Adding specific new definitions or sections to
+  files that have drifted, when updating the entire file isn't feasible.
+- **Resource updates**: Updating links to moved external resources.
```

Comment:
Sorry @vitorvasc! I must not have added this comment before submitting my review. I was thinking we could reorder this list so it matches more closely the headings that follow. 
 
```suggestion
- **Link maintenance**: Fixing broken link paths when pages are moved or
  deleted.
- **Resource updates**: Updating links to moved external resources.
- **Targeted content additions**: Adding specific new definitions or sections to
  files that have drifted, when updating the entire file isn't feasible.
```